plugins/rocks.xml.webhelp/xsl/dita2xmlrocksCoverImpl_template.xsl

```diff
@@ -3,6 +3,7 @@
 
   <xsl:import href="plugin:rocks.xml.webhelp:xsl/cover.xsl"/>
   <xsl:import href="plugin:rocks.xml.webhelp:xsl/dita2xmlrocks_common.xsl"/>
+  <xsl:import href="plugin:rocks.xml.webhelp:xsl/dita2xmlrocks_tables.xsl"/>
```

Comment:
DITA-OT has default filename for tables. We don't need 'dita2xmlrocks_' prefix here